gdc_client/parcel/utils.py

```diff
@@ -39,30 +37,26 @@ def check_transfer_size(actual, expected):
     return actual == expected
 
 
-def get_pbar(file_id, maxval, start_val=0):
+def get_pbar(file_id, maxval, start_val=0, desc="Downloading"):
     """Create and initialize a custom progressbar
 
-    :param str title: The text of the progress bar
-    "param int maxva': The maximumum value of the progress bar
+    Args:
+        file_id: file_id to include info about
+        maxval: maximum value for the progress bar
+        start_val: initial value for the progress bar
+        desc: display message next to the file_id
 
+    Returns:
+        tqdm: progress bar instance
     """
     log.debug("Downloading {0}:".format(file_id))
-    pbar = ProgressBar(
-        widgets=[
-            Percentage(),
-            " ",
-            Bar(marker="#", left="[", right="]"),
-            " ",
-            ETA(),
-            " ",
-            FileTransferSpeed(),
-            " ",
-        ],
-        initial_value=start_val,
-        maxval=maxval,
-        fd=sys.stdout,
+    pbar = tqdm(
```

Comment:
I don't remember but does the gdc-client use `#` to denote a progress bar? If so we can simulate that by using `ascii=True` as a parameter to tqdm. I anticipate this might come up during testing or user feedback.

Reply:
good point